src/EnergyPlus/AirflowNetwork/src/Elements.cpp

```diff
@@ -671,84 +633,55 @@ namespace AirflowNetwork {
         // REFERENCES:
         // na
 
-        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        Real64 CDM;
-        Real64 FL;
-        Real64 FT;
-        Real64 RhozNorm;
-        Real64 VisczNorm;
-        Real64 expn;
-        Real64 Ctl;
-        Real64 coef;
-        //Real64 Corr;
-        Real64 VisAve;
-        Real64 Tave;
-        Real64 RhoCor;
+        Real64 rhoz_norm = AIRDENSITY(StandardP, StandardT, StandardW);
+        Real64 viscz_norm = 1.71432e-5 + 4.828e-8 * StandardT;
 
-        // Formats
-        // static gio::Fmt Format_901("(A5,I3,6X,4E16.7)");
+        double VisAve{0.5 * (propN.viscosity + propM.viscosity)};
+        double Tave{0.5 * (propN.temperature + propM.temperature)};
```

Comment:
Looks like you've got a mix of `double` and `Real64`, obviously I'd prefer `Real64` so that the same bit-ness is assured throughout the code, but if you have a strong reason why AFN should use a raw `double` then I'm open to discussion.

Reply:
@Myoldmopar That's not intentional. Looks like I didn't commit the search and replace that swapped those out. Or maybe didn't push the commit. And we should just use double everywhere. 

I'll see what happened and push a fix.

Reply:
| And we should just use double everywhere.

I'm actually not fully opposed to that, but some folks like having a typedef that could potentially be altered all at once.  I already have a task for myself to propose changing the name to something else like `RealEP`.